docs/how-to/using-rocDecode-bitstream.rst

```diff
@@ -0,0 +1,215 @@
+.. meta::
+  :description: Using rocDecode with the bitstream decoder
+  :keywords: parse video, parse, decode, video decoder, video decoding, rocDecode, AMD, ROCm, bitstream decoder
+
+********************************************************************
+Using rocDecode with the bitstream decoder
+********************************************************************
+
+rocDecode core APIs are available in the |apifolder|_ of the `rocDecode GitHub repository <https://github.com/ROCm/rocDecode>`_. Utility classes that call the core APIs are available in the |utilsfolder|_ of the repository. For information about the core APIs, see :doc:`Using the rocDecode core APIs <./using-rocdecode>`.
+
+The `samples in the rocDecode GitHub repository <https://github.com/ROCm/rocDecode/tree/develop/samples>`_ use the utility classes. These classes are convenience classes that provide high-level calls to the core APIs. 
+
+A video stream can be decoded either with the rocDecode bitstream reader on GPU or the FFmpeg decoder on CPU. For information about using the FFmpeg decoder, see :doc:`Using rocDecode with the FFmpeg decoder <./using-rocDecode-ffmpeg>`.
+
+Import the ``roc_bitstream_reader.h`` and ``roc_video_dec.h`` header files. These headers contain the convenience classes and functions that use the core APIs.
+
+.. code:: C++ 
+
+    #include "roc_bitstream_reader.h"
+    #include "roc_video_dec.h"
+
+Use the ``rocDecCreateBitstreamReader`` to get a handle to a bitstream reader. The video file's bit depth and codec ID will be returned in the ``rocDecCreateBitstreamReader`` parameters.
+
+From the |videodecoderaw|_ sample:
```

Comment:
In the `From the |videodecoderaw|_ sample` section, please focus on explaining how the bitstream APIs are utilized in this sample. Do not include any information about the `RocVideoDecoder` utility class here. 

Additionally, we need to add a separate bullet point in the `How To Use` section, alongside the other bullet points, that conveys something similar to the following:

How to

- Use the rocDecode core APIs
- Use the rocDecode bitstream reader APIs
- Use the utility RocVideoDecoder class
- Use the utility FFMPEG demultiplexer
- Use the utility FFMPGE decoder